PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+Thanks for contributing to Apache Flink, before you open your PR please kindly take into consideration the following check list.
```

Comment:
Suggestion: Thanks for contributing to Apache Flink. Before you open your pull request, please take the following check list into consideration.